automated install/basic-install.sh

```diff
@@ -138,6 +138,45 @@ show_ascii_berry() {
 }
 
 # Compatibility
+setcap_check(){
+  # Print info regarding probing file capabilities
+  echo -e "  ${INFO} Testing for setting file capability in /usr/bin/"
+  # Create test file
+  ${SUDO} touch /usr/bin/pihole.setcap.test
```

Comment:
Can we have a temporary file in a non system directory for testing setcap? I don't like dropping a file, even temporarily, in to `/usr/bin/`. Should be able to test setcap on a file in `/tmp` even if the filesystem is set to noexec, since we aren't actually executing the file. Try using the random file generator function that is in the installer script and testing setcap on that file.